week1/homework/src/server.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
 
+const url = require('url');
 const http = require('http');
+// const fs = require('fs');
 
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
 function createServer(port) {
   let state = 10;
+  let newState = state;
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
-  });
+    const requestedUrl = url.parse(request.url, true);
```

Comment:
this doesn't hurt and it works fine, but you don't need to parse the url for this particular case,
`request.url` would work just fine :)